client/src/components/Feed/Post.js

```diff
@@ -84,7 +86,7 @@ const Post = ({
     }
   };
 
-  const loadComments = async () => {
+  const loadComments = useCallback(async () => {
```

Comment:
@joshmorel This one most likekly got added cuz of the `--fix`?

Reply:
Might not be an issue but I'd like to at least understand why it was eslint added it :thinking:

Reply:
Yeah I think it is because of this -> 
![image](https://user-images.githubusercontent.com/25408660/86047937-b3b4a600-ba1d-11ea-92a3-040152d67ec9.png)

Reply:
I mean this warning causes the trigger to --fix and it fixes it and put is back. I think is the way that some engineers use react hooks, it is not exactly the proper way?

Reply:
Or maybe not. I removed it and didn't get any eslint errors.  Any idea @cvaldez98 ?

Reply:
It complained about a missing useCallback dependency when I first added the dependency and the code was already there.